packages/deployment-service/src/resolve-production-s3-bucket-policies.ts

```diff
@@ -21,7 +12,11 @@ type BucketPolicyStatement = {
 
 const bucketArn = (bucketName: string): string => `arn:aws:s3:::${bucketName}`
 
-const resolveBucketPolicies = (client: S3Client) => async (bucketInputs: string[]) => {
+export const resolveProductionS3BucketPolicies = async () => {
+  const client = new S3Client({})
```

Comment:
docs make it look like you need to specify it here but guessing this uses the region that the lambda's running in?